docs(installation): update instructions for inertia (#3964)
nuxt/ui: docs/content/1.getting-started/2.installation/2.vue.md

This option adds the `transition-colors` class on components with hover or active states.
::

### `inertia`

> Enable the `inertia` option to ensure compatibility with [Inertia.js](https://inertiajs.com/).
> 
> ```ts [vite.config.ts]
> import { defineConfig } from 'vite'
> import vue from '@vitejs/plugin-vue'
> import ui from '@nuxt/ui/vite'
> 
> export default defineConfig({
>   plugins: [
>     vue(),
>     ui({
>       inertia: true
>     })
>   ]
> })
> ```
> 
> Note that `vue-router` is not required when using this option.

Reviewer: ```suggestion
Use the `inertia` option to ensure compatibility with [Inertia.js](https://inertiajs.com/).

```ts [vite.config.ts]
import { defineConfig } from 'vite'
import vue from '@vitejs/plugin-vue'
import ui from '@nuxt/ui/vite'

export default defineConfig({
  plugins: [
    vue(),
    ui({
      inertia: true
    })
  ]
})
```

::note
Note that `vue-router` is not required when using this option, this will use the `InertiaLink` component instead of `RouterLink`.
::
```
